conjur-api/Client.Methods.cs

```diff
@@ -24,6 +26,16 @@ public Variable Variable(string name)
             return new Variable(this, name);
         }
 
+		/// <summary>
+        /// Lists the variables.
+        /// </summary>
+        /// <returns>The variabßles.</returns>
```

Comment:
Typo. Also I feel the docstrings should be a little more detailed than just repeating what's in the method signature anyway.